src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -33,8 +33,16 @@ public static void BufferSize_GettingSameAsWindowSize()
     [Fact]
     public static void WindowWidth_WindowHeight_InvalidSize()
     {
-        Assert.Throws<ArgumentOutOfRangeException>("width", () => Console.WindowWidth = 0);
-        Assert.Throws<ArgumentOutOfRangeException>("height", () => Console.WindowHeight = 0);
+        if (Console.IsOutputRedirected)
```

Comment:
https://referencesource.microsoft.com/#mscorlib/system/console.cs,1093 Looking at the desktop source it looks like we should be throwing ArugmentOutOfRangeException when running on desktop as well. What am I missing?